front/components/vaults/VaultResourcesList.tsx

```diff
@@ -267,22 +332,35 @@ export const VaultResourcesList = ({
             />
           </div>
         )}
-        {!isSystemVault && category === "managed" && (
+        {!isSystemVault && isManaged && (
           <EditVaultManagedDataSourcesViews
             owner={owner}
             vault={vault}
             systemVault={systemVault}
             isAdmin={isAdmin}
           />
         )}
-        {(category === "folder" || category === "website") && (
-          <EditVaultStaticDataSourcesViews
-            owner={owner}
-            plan={plan}
-            vault={vault}
-            category={category}
-            dataSources={dataSources}
-          />
+        {isStatic && (
+          <>
+            <EditVaultStaticDatasourcesViews
+              isOpen={showFolderOrWebsiteModal}
+              setOpen={setShowFolderOrWebsiteModal}
+              owner={owner}
+              vault={vault}
+              dataSources={dataSources}
+              dataSourceView={selectedDataSourceView}
+              plan={plan}
+              category={category}
+              onClose={() => setSelectedDataSourceView(null)}
+            />
+            {selectedDataSourceView && (
+              <DeleteDataSourceDialog
```

Comment:
DeleteDataSourceDialog => DeleteStaticDataSourceDialog if it's only for static ( can't we deleted managed data sources ? )

Reply:
Renamed it ✅ 

IIRC we need to delete managed data sources through poke?